examples/create-react-app-with-typescript/package.json

```diff
@@ -12,7 +12,7 @@
     "typescript": "latest"
   },
   "scripts": {
-    "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
```

Comment:
Revert from #15765. Don't know why this was changed, @oliviertassinari?

Reply:
Was it introduced somewhere along with the v4 beta/alpha effort?

Reply:
It was introduced in the linked PR

Reply:
No, it wasn't: https://github.com/mui-org/material-ui/pull/15765/files#diff-359940529a1878d9a83b6c12574d522e. Don't let git trick you :).

Reply:
To answer your question, it should probably be reverted.

Reply:
I should probably have used the git move command.